tsunami/frontend/src/vdom.tsx

```
@@ -170,28 +170,46 @@ const SvgUrlIdAttributes = {
-            e.preventDefault();
+            e?.preventDefault?.();
         }
         if (fnDecl.stoppropagation) {
-            e.stopPropagation();
+            e?.stopPropagation?.();
         }
-        model.callVDomFunc(fnDecl, e, compId, propName);
+        let retVal: any;
+        if (fnDecl.jscode) {
+            try {
+                const fn = eval(fnDecl.jscode);
```

Biome (2.4.11)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)